src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -6,16 +6,16 @@
 import java.util.Properties;
 
 public class ConnectionUtil {
-    private static final String URL = "YOUR DATABASE URL";
-    private static final String USERNAME = "YOUR USERNAME";
-    private static final String PASSWORD = "YOUR PASSWORD";
-    private static final String JDBC_DRIVER = "YOUR DRIVER";
+    private static final String USERNAME = "root";
+    private static final String PASSWORD = "12345";
+    private static final String JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
+    private static final String URL = "jdbc:mysql://localhost:3306/taxi_service";
 
     static {
         try {
             Class.forName(JDBC_DRIVER);
         } catch (ClassNotFoundException e) {
-            throw new RuntimeException("Can't find SQL Driver", e);
+            throw new RuntimeException("Can't find SQL TaxiDatabase", e);
```

Comment:
```suggestion
            throw new RuntimeException("Can't find SQL Driver", e);
```
https://doc.qt.io/qt-5/sql-driver.html